examples/bpi-monolith-test-suite/bpi/components/zkmock/blockchain/shieldUtils.js

```diff
@@ -0,0 +1,103 @@
+import {
+  connectToBlockchain,
+  compileContract,
+  deployContract,
+  getDeployedShieldContractABI,
+  getDeployedShieldContractAddress,
+} from "./blockchain";
+import { fs } from "fs";
+
+const URL = "https://localhost:8080";
+
+export const deployShield = async () => {
+  var web3 = await connectToBlockchain();
+
+  //compile and deploy Verifier contract
+  var verifierContractAbsolutePath = path.resolve(
+    "../../zkcircuits/ordercircuit/order.sol"
+  );
+  var compiledVerifierContract = compileContract(
+    "Verifier.sol",
+    verifierContractAbsolutePath
+  );
+  var deployedVerifierContract = deployContract(
+    compiledVerifierContract,
+    web3.eth.defaultAccount
+  );
+
+  //compile and deploy Shield contract
+  var shieldContractAbsolutePath = path.resolve("./contracts/Shield.sol");
+  var compiledShieldContract = compileContract(
+    "Shield.sol",
+    shieldContractAbsolutePath
+  );
+  var deployedShieldContract = deployContract(
+    compiledShieldContract,
+    web3.eth.defaultAccount,
+    [deployedVerifierContract.options.address]
+  );
+
+  //Store the compiled Shield contract to access it later.
+  fs.writeFile("./contracts/Shield.json", compiledShieldContract, (err) => {
+    if (err) {
+      console.error(err);
+      return;
+    }
+    //file written successfully
+  });
+
+  //Store the address of deployed Shield contract
+  var shieldAddress = JSON.parse({
+    shieldAddress: deployedShieldContract.options.address,
+  });
+  fs.writeFile("./contracts/ShieldAddress.json", shieldAddress, (err) => {
```

Comment:
are all these generated files added to gitignore?

this file is also js, is this copied from somewhere else in codebase? if so, more argument to have them as util packages in future, we can maybe create github issues for this?